### PR TITLE
feat: Use Vue.js and a custom style for the viewer

### PR DESCRIPTION
**Pull request by MostafaBleu**
[Here is](https://codepen.io/mostafableu/full/PoojJPx) a codepen to play with the styling

### CODE REVIEW

**Comment by MostafaBleu** [comment]
[Here is](https://codepen.io/mostafableu/full/PoojJPx) a codepen to play with the new style

**Comment by charotAmine** [comment]
Hello, you also need to update the changelog so the users may see what was added.
Thanks.

File: src/preview.ts [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      if (artifactObj.properties.displayName) {
+        var nodeName = artifactObj.properties.displayName;
+      } else {
+        var filename = path.parse(filepath).base;
+        nodeName = path.parse(filename).name;
+      }
+
+      var found = false;
+      if (artifactObj.kind === "template") {
+        if (!artifactObj.properties.resourceGroup === undefined) {
```
[the remark this answers is not in the document]
Comment (MostafaBleu) [reply src/preview.ts]: I didn't change any logic inside this method it's the exact same existing code extracted to a separate file. 
This PR is focused on the Styling part of the viewer , I am planning to refactor the logic in a different PR

[the remark this answers is not in the document]
Comment (charotAmine) [reply src/preview.ts]: True. It was my bad. I am sorry adn thanks for fixing

**Pull request merged**